code/modules/client/client procs.dm

```diff
@@ -163,48 +177,62 @@
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Machine People?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Machine",15)
 							return
 					if("2")
 						if(karma <30)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Kidan?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Kidan",30)
 							return
 					if("3")
 						if(karma <30)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Grey?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Grey",30)
 							return
 					if("4")
 						if(karma <45)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Vox?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Vox",45)
 							return
 					if("5")
 						if(karma <45)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Slime People?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Slime People",45)
 							return
 					if("6")
 						if(karma <100)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Plasmaman?", "Confirmation", "Yes", "No") != "Yes")
+								return
 							DB_species_unlock("Plasmaman",100)
 							return
 					if("7")
 						if(karma <30)
 							to_chat(usr, "You do not have enough karma!")
 							return
 						else
+							if(alert("Are you sure you want to unlock Drask?", "Confirmation", "Yes", "No") != "Yes")
```

Comment:
You might want to switch the orders on these so that people don't accidentally tap space or enter and seal their own fate on accident